noir-projects/bootstrap_cache.sh

```diff
@@ -8,14 +8,13 @@ SCRIPTS_PATH=../build-system/s3-cache-scripts/
 echo -e "\033[1mRetrieving noir projects from remote cache...\033[0m"
 
 PROTOCOL_CIRCUITS_HASH=$(AZTEC_CACHE_REBUILD_PATTERNS="../noir/.rebuild_patterns_native ../barretenberg/cpp/.rebuild_patterns ./noir-protocol-circuits/.rebuild_patterns" $SCRIPTS_PATH/compute-content-hash.sh)
-$SCRIPTS_PATH/cache-download.sh noir-projects-noir-protocol-circuits-2-$PROTOCOL_CIRCUITS_HASH.tar.gz noir-protocol-circuits
-
 MOCK_CIRCUITS_HASH=$(AZTEC_CACHE_REBUILD_PATTERNS="../noir/.rebuild_patterns_native ../barretenberg/cpp/.rebuild_patterns ./mock-protocol-circuits/.rebuild_patterns" $SCRIPTS_PATH/compute-content-hash.sh)
-$SCRIPTS_PATH/cache-download.sh noir-projects-mock-protocol-circuits-$MOCK_CIRCUITS_HASH.tar.gz mock-protocol-circuits
-
 CONTRACTS_HASH=$(AZTEC_CACHE_REBUILD_PATTERNS="../noir/.rebuild_patterns_native ../avm-transpiler/.rebuild_patterns ../barretenberg/cpp/.rebuild_patterns noir-contracts/.rebuild_patterns" $SCRIPTS_PATH/compute-content-hash.sh)
-$SCRIPTS_PATH/cache-download.sh noir-projects-noir-contracts-$CONTRACTS_HASH.tar.gz noir-contracts
 
-yarn
+echo "
+noir-protocol-circuits
+mock-protocol-circuits
+noir-contracts
+" | xargs --max-procs 0 -I {} bash -c "$SCRIPTS_PATH/cache-download.sh noir-projects-{}-$HASH.tar.gz {}"
 
-(cd ./noir-protocol-circuits && yarn && node ./scripts/generate_variants.js)
```

Comment:
This is captured in cache now?

Reply:
Yes, the script creates the `crates/autogenerated` directory, the `private_kernel_reset_dimensions.json` file and root `Nargo.toml`. All three are part of the artifacts archive now :D